topics/ui/controls/checkbox.topic

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       id="checkbox" title="Checkbox">
+
+    <!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+    <title>
+        Checkbox
+    </title>
+    <link-summary>UI guidelines on using checkboxes.</link-summary>
+    <tldr>
+        <p>
+            <control>Implementation:</control>
+            <a href="https://docs.oracle.com/javase/tutorial/uiswing/components/button.html"><code>JCheckBox</code></a>,
+            <a href="%gh-ic%/platform/platform-api/src/com/intellij/ui/components/JBCheckBox.java"><code>JBCheckBox</code></a>
+        </p>
+    </tldr>
+    <img src="checkbox.png" alt="A preview of checkboxes in different states" width="706"/>
+    <chapter title="When to use" id="when-to-use">
+        <p>Use checkboxes for yes/no choices or for selecting several items in a group:</p>
+        <img src="checkbox_when_to_use.png" alt="An example of a checkbox group with two selected options" width="706"/>
+    </chapter>
+    <chapter title="When not to use" id="when-not-to-use">
+        <p>Use <a href="radio_button.md">radio button</a> instead if:</p>
+        <table style="none" border="false">
+            <tr>
+                <td width="50%">
+                    <img src="checkbox_when_not_to_use_1.png" alt="A radio button group with only one selected option possible" width="378"/>
+                </td>
+                <td>Only one option in a group can be selected</td>
+            </tr>
+            <tr>
+                <td width="50%">
+                    <img src="checkbox_when_not_to_use_2.png" alt="Two radio buttons with clear labels" width="378"/>
+                </td>
+                <td>The behavior in the "off" state is unclear from the checkbox label. Use two radio buttons instead
+                    and label them accordingly
+                </td>
+            </tr>
+            <tr>
+                <td width="50%">
+                    <img src="checkbox_when_not_to_use_3.png" alt="A radio button group displaying the correct way for showing different states of a setting" width="378"/>
+                </td>
+                <td>With the checkbox, it is unclear how the setting works if it’s unchecked. With radio buttons, both
```

Comment:
Looks like this rule repeats the previous one. Maybe merge the text and leave only one image?